sdk/triggers.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const { getTriggersListUrl, getSpecificTriggerUrl } = require('./url');
+const { getValidatedBody } = require('./handleError');
+
+async function upsert(apiKey, triggerName, triggerOptions) {
+  const listOptions = {
+    url: getSpecificTriggerUrl(triggerName),
+    headers: {
```

Comment:
This can be defined once for all trigger actions

Reply:
👍